sys/net/if.c

```diff
@@ -3061,14 +3057,14 @@ ifioctl(struct socket *so, u_long cmd, caddr_t data, struct thread *td)
 #endif
 
 #if defined(COMPAT_FREEBSD32) || defined(COMPAT_FREEBSD64)
+	saved_cmd = cmd;
+	saved_data = data;
```

Comment:
With the change to use (e.g.) `ifc32` directly later on are we using `saved_data` at all anymore?